tests/utils/dbUtils.ts

```diff
@@ -0,0 +1,64 @@
+import { exec } from 'child_process';
+import { Umzug, SequelizeStorage, MigrationError } from 'umzug';
+import { Sequelize } from 'sequelize'
+import path from 'path';
+import db from '../../src/models';
+import { calledFromTestFileOrDirectory } from './testOnly';
+
+const clear = async () => {
+  await db.sequelize.query(`
+    DROP SCHEMA public CASCADE;
+    CREATE SCHEMA public;
+  `);
+};
+
+const loadMigrations = async (migrationSet:string): Promise<void> => {
+  const migrationPattern = '*.js'; // File extension pattern for migration files
+  const migrationDir = `src/${migrationSet}/${migrationPattern}`; // path.join('./', migrationSet, migrationPattern);
+
+  console.log(migrationDir);
+
+  const umzug = new Umzug({
+    storage: new SequelizeStorage({ sequelize: db.sequelize }),
+    migrations: {
+      glob: migrationDir,
+      resolve: ({ name, path, context }) => {
+        const migration = require(path);
+        return {
+            name,
+            up: async () => migration.up(context, Sequelize),
+            down: async () => migration.down(context, Sequelize),
+        };
+      },
+    },
+    context: db.sequelize.getQueryInterface(),
+    logger: console,
+  });
+
+  try {
+    const migrations = await umzug.up();
+    console.log(`Successfully executed ${migrations.length} migrations.`);
```

Comment:
I think these need to be passed to the auditLogger to show up in certain CI jobs